docs/UsersGuide/source/Components.rst

```diff
@@ -0,0 +1,81 @@
+SRW Application Components
+============================
+
+The SRW Application assembles a variety of components, including:
+
+* Pre-processor Utilities & Initial Conditions
+* UFS Weather Forecast Model
+* Unified Post-Processor
+* Visualization Examples
+* Build System and Workflow
+
+These components are documented within this User's Guide and supported through a `community forum <https://forums.ufscommunity.org/>`_. 
+
+.. _Utils:
+
+Pre-processor Utilities and Initial Conditions
+==============================================
+
+The SRW Application includes a number of pre-processing utilities that initialize and prepare the model. Since the SRW App provides forecast predictions over a limited area (rather than globally), it is necessary to first generate a regional grid (``regional_esg_grid/make_hgrid``) along with :term:`orography` (``orog``) and surface climatology (``sfc_climo_gen``) files on that grid. Grids include a strip, or "halo," of six cells that surround the regional grid and feed in lateral boundary condition data. Since different grid and orography files require different numbers of halo cells, additional utilities handle the correct number of halo shave points and topography filtering. The pre-processing software ``chgres_cube`` is used to convert the raw external model data into initial and lateral boundary condition files in netCDF format. These are needed as input to the FV3-LAM. Additional information about the UFS pre-processor utilities can be found in the `UFS_UTILS User’s Guide <https://noaa-emcufs-utils.readthedocs.io/en/ufs-v2.0.0/>`_.
+
+The SRW Application can be initialized from a range of operational initial condition files. It is possible to initialize the model from the Global Forecast System (:term:`GFS`), North American Mesoscale (:term:`NAM`) Forecast System, Rapid Refresh (:term:`RAP`), and High-Resolution Rapid Refresh (:term:`HRRR`) files in Gridded Binary v2 (:term:`GRIB2`) format. GFS files also come in :term:`NEMSIO` format for past dates. 
+
+.. WARNING::
+   For GFS data, dates prior to 1 January 2018 may work but are not guaranteed. Public archives of model data can be accessed through the `National Centers for Environmental Information <https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_ (NCEI) or through the `NOAA Operational Model Archive and Distribution System <https://nomads.ncep.noaa.gov/>`_ (NOMADS). Raw external model data may be pre-staged on disk by the user.
+
+
+Forecast Model
+==============
+
+The prognostic atmospheric model in the UFS SRW Application is the Finite-Volume Cubed-Sphere
+(:term:`FV3`) dynamical core configured with a Limited Area Model (:term:`LAM`) capability :cite:`BlackEtAl2020`. The dynamical core is the computational part of a model that solves the equations of fluid motion. A User’s Guide for the UFS :term:`Weather Model` is `here <https://ufs-weather-model.readthedocs.io/en/ufs-v2.0.0/>`__. 
```

Comment:
Same comment for all these components applies - will these versions be updated for this release of the SRW App? I would think there would be quite a few changes possibly needed for the UFS WM as it has changed a lot in a year so it likely should be updated to v3.0.0.

Reply:
I'm still seeing only a v2.0.0 tag on the WM repo and in their docs. I've changed the docs to represent "latest" in most cases but can change them at a later date to a specific version for the SRW App release.

Reply:
I think it would be good to make sure someone is working on the updated documentation for the WM UG. It will need to be tagged for the new release.

Reply:
I think it would be good to check and make sure that the WM UG is being updated and there is a plan to tag something for the v3 release. I am not sure who is coordinating that.